tools/lint/lint.py

```diff
@@ -995,6 +1026,10 @@ def process_errors(errors):
             assert logger is not None
             for line in (ERROR_MSG % (last[0], last[1], last[0], last[1])).split("\n"):
                 logger.info(line)
+
+    if error_count and github_checks_outputter:
+        github_checks_outputter.output("```")
```

Comment:
Can we move this to the end of `process_errors`? If I'm reading this correctly, we don't add any other checks output between the end of `process_errors` and here; if so, I'd like them to be closer.

Reply:
I don't quite follow what you mean - we potentially call `process_errors` many times, once per error type (see lines 1012, 1017, and 1020).

Reply:
Oh I did not fully expand the file... I was just hoping we could encapsulate this better / in a single place. Since `process_errors` is called multiple times, we might need something like registering a prefix / suffix in GitHubChecksOutputter when instantiated so that we don't need to weave the output text in the lint logic, but that might be over-engineering. Feel free to ignore.